src/index.css

```diff
@@ -77,3 +80,165 @@ form {
   display: flex;
   justify-content: space-between;
 }
+
+.add-btn {
+  background-color: #0290ff;
+  color: #fff;
+  font-size: 20px;
+  border: none;
+  height: 40px;
+  width: 150px;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.input-text {
+  width: 30%;
+  border-radius: 5px;
+  border: 1px solid gray;
+  padding-left: 15px;
+  font-size: 18px;
+  font-weight: 200;
+}
+
+.action-btn {
+  font-family: 'Roboto Slab', serif;
+  background-color: transparent;
+  border: none;
+  color: #4386bf;
+  font-size: 0.875rem;
+  font-weight: 300;
+  align-items: center;
+  cursor: pointer;
+}
+
+.remove-btn {
+  border-left: 2px solid #e8e8e8;
+  border-right: 2px solid #e8e8e8;
+}
+
+.prog-btn {
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  letter-spacing: 0.5px;
+  color: #fff;
+  border: none;
+  height: 35px;
+  width: 180px;
```

Comment:
- [OPTIONAL] I noticed that you repeated some of the CSS _**declarations**_ across many **_selectors_**. Declarations like `display: flex;`, `font-family: 'Roboto Slab', serif;` and many others. In CSS, one of the good practices is to avoid repeating the same **_rule/declarations_** across **_many selectors (classes, elements)_**. I would advise that you put all _**rules/declarations**_ that are repeated across several elements in a class and then use the class as a `className` on the desired JSX in the component. To understand better what I am driving at, kindly read this [article](https://speckyboy.com/good-bad-css-practices/). 👍 💻